suites/pacific/rbd/tier_1_rbd.yaml

```diff
@@ -0,0 +1,168 @@
+# Tier1: Extended RBD acceptance testing
+#
+# This test suite runs addition test scripts to evaluate the existing functionality of
+# Ceph RBD component.
+#
+# The following testing is carried out
+#   - verification of NBD unmap/map
```

Comment:
not to nitpick, but there is a type here. i think it should be `RBD` instead of `NBD`  :)

Reply:
My bad, @psathyan clarified. it's `NBD - network block device. `